packages/app-cli/tests/models_Tag.ts

```diff
@@ -123,21 +133,25 @@ describe('models_Tag', function() {
 		expect(commonTags.length).toBe(0);
 
 		commonTags = await Tag.commonTagsByNoteIds([note0.id, note1.id, note2.id, note3.id]);
+		// @ts-ignore
```

Comment:
Why do we need ts-ignore here?

Reply:
I can't figure out why I added the ts-ignore, I have removed them.